src/keygen/keyshare.rs

```diff
@@ -42,6 +49,91 @@ impl KeySharePrivate {
     pub(crate) fn public_share(&self) -> Result<CurvePoint> {
         CurvePoint::GENERATOR.multiply_by_scalar(&self.x)
     }
+
+    /// Convert private material into bytes.
+    ///
+    /// 🔒 This is intended for use by the calling application for secure
+    /// storage. The output of this function should be handled with care.
+    pub fn into_bytes(self) -> Vec<u8> {
+        // Format:
+        // KEYSHARE_TAG | key_len in bytes | key (big endian bytes)
+        //              | 8 bytes          | key_len bytes
+
+        let mut share = self.x.to_bytes();
+        let share_len = share.len().to_le_bytes();
+
+        let bytes = [KEYSHARE_TAG, &share_len, &share].concat();
+        share.zeroize();
+        bytes
+    }
+
+    /// Convert bytes into private material.
+    ///
+    /// 🔒 This is intended for use by the calling application for secure
+    /// storage. Do not use this method to create arbitrary instances of
+    /// [`KeySharePrivate`].
+    pub fn try_from_bytes(bytes: Vec<u8>) -> Result<Self> {
+        // Expected format:
+        // KEYSHARE_TAG | key_len in bytes | key (big endian bytes)
+        //              | 8 bytes          | key_len bytes
+
+        let mut parser = ParseBytes::new(bytes);
+
+        // This little method ensures that
```

Comment:
Tiny nit, "method" implies there is an object associated with the function. I would use function, closure or lambda.